client/login/desktop-login/start.tsx

```diff
@@ -0,0 +1,48 @@
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import FormattedHeader from 'calypso/components/formatted-header';
+import Main from 'calypso/components/main';
+import { Notice } from 'calypso/components/notice';
```

Comment:
Related to the `translate` prop: if you import the default import from the same file:
```js
import Notice from 'calypso/components/notice';
```
then that will be the version wrapped with `localize`. That's insidious, I'm going to fix that and export only one version.

Reply:
Thanks for opening that PR to fix this, it was indeed confusing to me.